azure/azure-pr-pipeline.yml

```diff
@@ -58,7 +64,7 @@ extends:
           post_deploy:
             - template: templates/pds-tests.yml
           depends_on:
-            - karate_tests
+            - karate_sandbox_tests
```

Comment:
Don't know much about azure, but is there any issues with running the karate stages in parallel?